provider_client.go

```diff
@@ -97,7 +124,6 @@ func (client *ProviderClient) Request(method, url string, options RequestOpts) (
 
 	// Derive the content body by either encoding an arbitrary object as JSON, or by taking a provided
 	// io.Reader as-is. Default the content-type to application/json.
-
```

Comment:
Yeah, I don't know. All the comments had a blank line between them and the code on which they commented.